examples/website-demos/social-messaging/src/components/ChannelPreview/SocialChannelPreview.tsx

```diff
@@ -28,12 +31,18 @@ export const SocialChannelPreview: React.FC<ChannelPreviewUIComponentProps> = (p
     const members = Object.values(channel.state.members).filter(
       ({ user }) => user?.id !== client.userID,
     );
+
+    const handleUnreadCounts = () => {
+      if (unread && unread > 0) setChatsUnreadCount(chatsUnreadCount - unread);
```

Comment:
can you just do this in both conditionals? since 0 is falsey
```suggestion
      if (unread) setChatsUnreadCount(chatsUnreadCount - unread);
```

Reply:
unread could be undefined though, so we need to check to see if it exists.

Reply:
yeah but if undefined it still won't enter into that conditional, it should perform the same either way, right?

Reply:
TypeScript doesn't like it

<img width="1240" alt="Screen Shot 2021-08-02 at 3 18 53 PM" src="https://user-images.githubusercontent.com/18018191/127930865-d40b7f74-fd3c-4e45-9e11-a03de01e4840.png">

Reply:
i mean just `if (unread)`, not > 0

Reply:
Oh, I misread what you suggested up there initially!
But, I still want to check if it's greater than 0. I was actually logging that number as zero and though the count would be correct (if not checking for greater then and then subtracting 0), I'd rather not run the that state setting calculation. Or, does that matter if you're trying to set state as the same number?

Reply:
Ohhhhh, disregard - I see. 0 is falsey. Ok, I will update to if (unread) once I'm back on my computer!

Reply:
haha yep that's what i was getting at, nothing like talking past each other in GH comments lol